vendor/github.com/redhat-cne/sdk-go/pkg/subscriber/subscriber_reader.go

```diff
@@ -31,9 +31,8 @@ func (s *Subscriber) GetSubStore() *store.PubSubStore {
 
 // CreateCloudEvents ...
 func (s *Subscriber) CreateCloudEvents() (*cloudevents.Event, error) {
-	ce := cloudevents.NewEvent(cloudevents.VersionV03)
-	ce.SetDataContentType(cloudevents.ApplicationJSON)
-	ce.SetSpecVersion(cloudevents.VersionV03)
+	ce := cloudevents.NewEvent(cloudevents.VersionV1)
+	ce.SetSpecVersion(cloudevents.VersionV1)
```

Comment:
why not setting content type?

Reply:
In O-RAN V3.0 spec, Event Data Model does not have `datacontenttype` and `subject`, so I removed them.

Reply:
Subject is used  by cloudEvent proxy  it is internally used 
https://github.com/redhat-cne/cloud-event-proxy/blob/d9e98a115ad764a2d15a4ab5b36594b2228d333c/examples/simplehttp/common/subscribe.go#L48

Reply:
also it is used by cloudevents specs https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#example

Reply:
removed ce.SetSubject() from simplehttp. I think we are fine here. The subject is not actually used in internal API. The ProcessOutChannel() only uses `Status` and `Action`.
https://github.com/redhat-cne/cloud-event-proxy/blob/main/cmd/main.go#L219